tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -127,3 +127,37 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario: upload a file with virus to a spaces inside the folder
+    Given using spaces DAV path
+    And the administrator has given "Alice" the role "Space Admin" using the settings api
+    And user "Alice" has created a space "new-space" with the default quota using the GraphApi
+    And user "Alice" has created a folder "uploadFolder" in space "new-space"
+    When user "Alice" uploads a file inside space "new-space" with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to "/uploadFolder/virusFile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Alice" should get a notification with subject "Virus found" and message:
+      | message                                                                        |
+      | Virus found in virusFile.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Alice" file "/uploadFolder/virusFile.txt" should not exist
+    When user "Alice" uploads a file inside space "new-space" with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to "/uploadFolder/virusFile.txt" using the WebDAV API
```

Comment:
duplicate step? you probably wanted to upload the file somewhere else?

Reply:
@nabim777 yes this can be `/uploadFolder/virusFile.txt` but change the step above to upload on root level `/virusFile.txt`. Check my review

Reply:
oh sorry I forget to edit the path

Reply:
Oh my mistake !!!! 
I forget to edit the path